docs/meshstack.billing-configuration.md

```diff
@@ -511,40 +511,28 @@ let example
 
 You have two options to enable currency conversion for chargebacks and usage reports:
 
-- For the all Plans default converter is available. For the default converter, we are getting rates from [frankfurter.app](https://www.frankfurter.app/), which publishes exchange rates by the European Central Bank.Go to the Financial Tab on the Settings page and simply activate a toggle to use the default converter. 
-- If you are on the Power Plan, you can integrate your own converter via API. Simply turn off the toggle button in the Financial Tab of the Settings page and reach out to our support team. They will support you in connecting your preferred converter with meshStack.
+1. **Default converter** fetches rates from [frankfurter.app](https://www.frankfurter.app/), which publishes exchange rates from the European Central Bank. To enable the default converter, simply activate the toggle in the Financial Tab on the Settings page
+2. If your organization has its own rates, you can utilize **Own converter**, which sends rates to meshStack via API. Simply turn off the toggle button in the Financial Tab of the Settings page and reach out to our support team support@meshcloud.io. They will support you in connecting your own converter with meshStack.
 
-If your meshStack is hosted in a private environment and has permission to make external requests, you can activate the default converter using the toggle button on the Settings page, as mentioned earlier. If external requests are restricted, you have the option to enable meshStack to communicate with the Frankfurter API through Firewall rules. Alternatively, if no conversion is desired for chargebacks, you can choose not to apply any conversion by simply turning off togle button in the Financial Tab of the Settings page.
+For chargebacks, we use the daily exchange rate from the day your statement is finalized, regardless of whether you're using your own converter or the default one. For example,  your chargeback finalization day is March 31st, and the USD to Euro rate on that day is 0.93, then all March chargebacks in USD will be converted and finilized at this rate.
 
-> Please be aware that enabling the default converter or integrating your own one will only affect data moving forward, namely past chargebacks and usage reports won´t retroactively convert to the base currency. 
+> Please be aware that activating the default converter or adding your own will only impact future data. Past finalized chargebacks and usage reports won't be retroactively converted to the single currency. So, if you want currency conversion for the current month, make sure to connect or activate it before finalizing chargebacks for that month
```

Comment:
```suggestion
> Please be aware that activating the default converter or adding your own will only impact future data. Past finalized chargeback statements and usage reports won't be retroactively converted to the single currency. So, if you want to enable currency conversion for the current month, make sure to connect or activate it before finalizing chargebacks for that month.
```
This might also change